addons/glusterfs/docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM gluster/glusterfs-client:latest
```

Comment:
Please specify the image version to make sure it's reproducible.

Reply:
Image "gluster/glusterfs-client" only has the latest tag on docker hub.

Reply:
You can use DIGEST. Just like gluster/glusterfs-client@sha256:66b1d51d327ab1c4c1d81e6bad28444e13e1746c2d6f009f9874dad2fba9836e

Reply:
Get it